app/backend/src/couchers/email/__init__.py

```diff
@@ -95,11 +95,15 @@ def queue_email(sender_name, sender_email, recipient, subject, plain, html):
 
 def enqueue_email_from_template(recipient, template_file, template_args={}):
     frontmatter, plain, html = _render_email(template_file, template_args)
+    if not config["NOTIFICATION_EMAIL_PREFIX"]:
+        subject = frontmatter["subject"]
+    else:
+        subject = config["NOTIFICATION_EMAIL_PREFIX"] + " " + frontmatter["subject"]
```

Comment:
```suggestion
    subject = config["NOTIFICATION_EMAIL_PREFIX"] + " " + frontmatter["subject"]
```

and include the space in NOTIFICATION_EMAIL_PREFIX.

Reply:
I just thought it'd be easier this way since trailing spaces can be a bit finicky with env vars? But I can change it

Reply:
Ah true, but not the end of the world if the space is omitted, it is only for testing.

Reply:
Good point, I'll change it

Reply:
Done!